osu.Game/Screens/Edit/Editor.cs

```diff
@@ -630,9 +630,7 @@ private void refetchBeatmap()
             // To update the game-wide beatmap with any changes, perform a re-fetch on exit/suspend.
             // This is required as the editor makes its local changes via EditorBeatmap
             // (which are not propagated outwards to a potentially cached WorkingBeatmap).
-            ((IWorkingBeatmapCache)beatmapManager).Invalidate(Beatmap.Value.BeatmapInfo);
-            var refetchedBeatmapInfo = beatmapManager.QueryBeatmap(b => b.ID == Beatmap.Value.BeatmapInfo.ID);
```

Comment:
Just to confirm intent: the idea here was that the `.Save()` call results in a `BeatmapUpdater.Process()` call, which does all of the invalidating this method used to, and then updates the value of `Beatmap.Value` to the correct post-save-and-invalidation one? Removing the refetch from realm here initially spooked me but if the idea above is what was intended, then I think it's fine (as long as the test failures are sorted).

Reply:
The refetch is actually done via the `bool` parameter now.